Resources/Textures/Objects/Weapons/Melee/wardenshield.rsi/meta.json

```diff
@@ -0,0 +1,33 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Sprite's made by minthydra",
```

Comment:
Reference other meta.json files to make sure this is consistent. At the least `Sprites` rather than `Sprite's`.